keps/sig-node/1769-memory-manager/README.md

```diff
@@ -586,20 +658,34 @@ the health of the service?**
     To understand the reason you will need to check via pod resources API 
     the amount of allocatable memory and memory reserved by containers.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
   This does not seem relevant to this feature.
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+<!--
   Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
   implementation difficulties, etc.).
+-->
   Currently, for the pod author, it is impossible to know containers NUMA pinning without access to the node.
 
 ### Dependencies
 
 _This section must be completed when targeting beta graduation to a release._
 
-* **Does this feature depend on any specific services running in the cluster?**
+###### Does this feature depend on any specific services running in the cluster?
```

Comment:
The scalability section below is mandatory for GA and is not populated. Can we populate it? Thanks!

Reply:
The scalability appears right below. line 692